app/controllers/users_controller.rb

```diff
@@ -14,6 +14,11 @@ def create
     end
   end
 
+  def email_taken
```

Comment:
@7imbrook it can be accessed authed or not authed. Changed it from a post to a get with email as a query param